src/Pelagos/Util/UrlValidation.php

```diff
@@ -20,8 +20,12 @@ class UrlValidation
     public function validateUrl($url)
     {
         $ch = curl_init();
+        $cookies = tempnam('/tmp','cookie.txt');
+        curl_setopt($ch, CURLOPT_COOKIEJAR, $cookies);
+        curl_setopt($ch, CURLOPT_COOKIEFILE, $cookies);
         curl_setopt($ch, CURLOPT_URL, $url);
         curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
         $output = curl_exec($ch);
```

Comment:
$output never used. Also look into CURLOPT_NOBODY which does a HEAD instead of GET.

Reply:
Changed it.